examples/advanced/tidb-cluster.yaml

```diff
@@ -1,78 +1,278 @@
+    # maxFailoverCount: 3
+
+    ## describes the compute resource requirements and limits.
+    ## Ref: https://kubernetes.io/docs/concepts/configuration/manage-compute-resources-container/
     requests:
-      cpu: "100m"
+      cpu: 100m
+      memory: 256Mi
       storage: 1Gi
+    limits:
+      cpu: 1000m
+      memory: 2Gi
+      storage: 10Gi
 
-    # TiDB Operator will automatically mount the certificates
-    # Defaults to "false"
-    # Ref: https://docs.pingcap.com/tidb-in-kubernetes/stable/configure-a-tidb-cluster#mountclusterclientsecret
+    ## define Kubernetes service for pd-server
+    ## Ref: https://kubernetes.io/docs/concepts/services-networking/service/
+    # service:
+    #   type: ClusterIP
+    #   annotations:
+    #     foo: bar
+    #   loadBalancerIP: load-balancer-ip
+    #   clusterIP: cluster-ip
+    #   portName: 2379
+
+    #############################
+    # Advanced PD Configuration #
+    #############################
+
+    ## Overwrite TiDB cluster level configurations
+    # version: "v4.0.8"
+    # imagePullPolicy: IfNotPresent
+    # ImagePullSecrets: secretName
+    # hostNetwork: false
+    # serviceAccount: advanced-tidb-pd
+    # priorityClassName: system-cluster-critical
+    # schedulerName: tidb-scheduler
+    # nodeSelector:
+    #   app.kubernetes.io/component: pd
+    # annotations:
+    #   node.kubernetes.io/instance-type: some-vm-type
+    # tolerations:
+    #   - effect: NoSchedule
+    #     key: dedicated
+    #     operator: Equal
+    #     value: pd
+    # configUpdateStrategy: RollingUpdate
+    # statefulSetUpdateStrategy: RollingUpdate
+
+    ## List of environment variables to set in the container
+    ## Note that the following env names cannot be used and may be overrided by TiDB Operator builtin envs
+    ##   - NAMESPACE
+    ##   - TZ
+    ##   - SERVICE_NAME
+    ##   - PEER_SERVICE_NAME
+    ##   - HEADLESS_SERVICE_NAME
+    ##   - SET_NAME
+    ##   - HOSTNAME
+    ##   - CLUSTER_NAME
+    ##   - POD_NAME
+    ##   - BINLOG_ENABLED
+    ##   - SLOW_LOG_FILE
+    ## Ref: https://kubernetes.io/docs/tasks/inject-data-application/environment-variable-expose-pod-information/
+    # env:
+    #   - name: MY_ENV_1
+    #     value: value1
+    #   - name: MY_ENV_2
+    #     valueFrom:
+    #       fieldRef:
+    #         fieldPath: status.myEnv2
+
+    ## Custom sidecar containers can be injected into the PD pods,
+    ## which can act as a logging/tracing agent or for any other use case
+    # additionalContainers:
+    #   - name: myCustomContainer
+    #     image: ubuntu
+
+    ## custom additional volumes in PD pods
+    # additionalVolumes:
+    #   - name: nfs
+    #     nfs:
+    #       server: 192.168.0.2
+    #       path: /nfs
+
+    ## custom additional volume mounts in PD pods
+    # additionalVolumeMounts:
+    #   - name: nfs
+    #     mountPath: /nfs
+
+    ## define the time period between pod Terminating and killed
```

Comment:
Could you please take the descriptions from the k8s code?
Actually, many fields are referred to the k8s definition, please also take their descriptions.